src/Core/OOMusicController.m

```diff
@@ -166,6 +166,12 @@ - (void) playMissionMusic
 }
 
 
+- (void) setMusicGain:(float)gain
+{
+	[_current setMusicGain:gain];
```

Comment:
Bounds checking? I believe this could be something like `[_current setMusicGain:OOClamp_0_1_f(gain)]`

Reply:
Oh yeah, makes sense, fixed it now.